services/kuber/server/scrapeConfig/scrapeConfig.go

```diff
@@ -0,0 +1,73 @@
+package scrapeconfig
+
+import (
+	"fmt"
+
+	"github.com/Berops/claudie/internal/kubectl"
+	"github.com/Berops/claudie/internal/templateUtils"
+	"github.com/Berops/claudie/proto/pb"
+)
+
+type ScrapeConfig struct {
+	Cluster    *pb.K8Scluster
+	LBClusters []*pb.LBcluster
+	Directory  string
+}
+
+type SCData struct {
+	LBClusters []*pb.LBcluster
+}
+
+type NSData struct {
+	Namespace string
+}
+
+const (
+	namespaceFileTpl      = "namespace.goyaml"
+	namespaceFile         = "namespace.yaml"
+	scrapeConfigNamespace = "monitoring"
+	scrapeConfigName      = "additional-scrape-config"
+	scrapeConfigFileTpl   = "scrape-config.goyaml"
+	scrapeConfigFile      = "scrape-config.yaml"
+)
+
+// GenerateAndApplyScrapeConfig will template a secret with Prometheus configuration,
+// for scraping Loadbalancers node-exporter endpoints
+// it will create the secret in a applied namespace
+// If there is no loadbalancers it will apply the config with no target endpoints
```

Comment:
If there are no LBs, is it necessary to apply an empty config?

Reply:
In situation when a cluster changes and removes the LB cluster, I wanted to do a simple approach with just applying empty config, without adding additional logic for removing old configs. If you do not like it I can try to change it

Reply:
Done as you advised